keras/backend/theano_backend.py

```diff
@@ -3,7 +3,6 @@
 from theano.sandbox.rng_mrg import MRG_RandomStreams as RandomStreams
 from theano.tensor.signal import pool
 from theano.tensor.nnet import conv3d2d
-from theano.sandbox import softsign as T_softsign
```

Comment:
Please use `try: from theano.tensor.nnet.nnet import softsign`, `except` for backwards compatbility with previous versions of Theano.